salt/modules/reg.py

```diff
@@ -747,3 +747,56 @@ def delete_value(hive, key, vname=None, use_32bit_registry=False):
         log.error('ValueName: {0}'.format(local_vname))
         log.error('32bit Reg: {0}'.format(use_32bit_registry))
         return False
+
+
+def regfile_import(source, use_32bit_registry=False):
```

Comment:
To maintain continuity with other function names in this module could we call this `import_file`?